scss/civicrm/common/_buttons.scss

```diff
@@ -59,6 +59,52 @@ $button-border-radius: 2px;
     }
   }
 
+  input.crm-form-submit {
```

Comment:
do you need to specify `input`? Can't it be just `.crm-form-submit` ?

Reply:
We need this as civicrm.css from main theme has default style with same input.crm-form-submit and that prevents us from overriding style if that's just `.crm-form-submit`